services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -437,6 +440,23 @@ public void routeOptionsWithUTF8Chars_toUrlWithEncodedChars() {
     assertEquals(expectedEncodedUrl, url.toString());
   }
 
+  @Test
+  public void routeOptionWithExcludedWaypoint() {
+    RouteOptions options = RouteOptions.builder()
+        .coordinates("1.0,1.0;2.0,2.0")
+        .profile(DirectionsCriteria.PROFILE_DRIVING)
+        .excludeList(Collections.singletonList(
```

Comment:
let's add separate property to RouteOptions, like `excludePointsList(List<Point> excludePointsList)` and process exclude profiles and coordinates separately

Reply:
that true, that each RouteOptions field corresponded to request parameter, but previously parameters weren't mixed in the same place. 

Where it's actually mixed is URL, which we handle under the hood. So it's not a problem for customers I think